test/bin/grub-editenv

```diff
@@ -69,6 +86,26 @@ function variable_set {
 			echo "Invalid argument for B_TRY: '$2'"
 			exit 1
 		fi
+	elif [ "$1" = "system0_OK" ]; then
```

Comment:
Are these names used in any of our tests?

Reply:
Definintely used/needed for the `rauc.t` `install *` tests. I think the `install.c` unit-test also uses it, but not sure.

These changes should also be usable with the qemu test, if one uses the script instead of the real `grub-editenv`